examples/js/loaders/PlayCanvasLoader.js

```diff
@@ -140,7 +140,7 @@ THREE.PlayCanvasLoader.prototype = {
 			object.name = data.name;
 
 			object.position.fromArray( data.position );
-			object.rotation.fromArray( data.rotation );
+			object.quaternion.setFromEuler( new THREE.Euler().fromArray( data.rotation ) );
```

Comment:
@Mugen87 Again, instantiating a new instance may be OK...

Reply:
:+1: